goss.yaml

```diff
@@ -0,0 +1,22 @@
+process:
+  otelcol:
+    running: true
+
+user:
+  _daemon_:
+    exists: true
+    uid: 584792
+    groups:
+    - _daemon_
+
+file:
+  /etc/otelcol/config.yaml:
+    exists: true
+    mode: "0644"
+    owner: root
+    group: root
```

Comment:
This combo seems odd:
- owner is root but 0644 implies `o+r`; and
- the service is run as non-root, so able to read thanks to the `o+r`.

Not sure what the best practice is. Upstream has this:

![image](https://github.com/user-attachments/assets/80588cc1-0d61-4ec7-a001-1c52dc3dd4b1)

Reply:
upstream has: `-rw-r--r--` and we have the same:


```shell
_daemon_@0997a5f7f341:/$ ls -lh /etc/otelcol/config.yaml
-rw-r--r-- 1 root root 1.5K Jan  8 13:40 /etc/otelcol/config.yaml
```

About the owner and group, do you think it should be other than `root`?